### PR TITLE
Add picture, publish to post

### PR DESCRIPTION
**Pull request by thanhgiong195**

### CODE REVIEW

**Comment by thanhgiong195** [comment]
ready

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -1,16 +1,26 @@
 <div class="wthree">
   <div class="col-md-6 wthree-left wow fadeInDown">
     <div class="tch-img">
-      <a href="#"><%= image_tag "f1.jpg", class: "img-responsive" %></a>
+      <div class="img-post">
+        <% if post.picture? %>
```
Comment (huongnguyenmta) [inline app/views/posts/_post.html.erb]: Trong `app/uploaders/picture_uploader.rb` em định nghĩa `default_url` cho picture đi nhé

**Comment by huongnguyenmta** [comment]
commented

**Comment by thanhgiong195** [comment]
ready

File: db/migrate/20170627063512_create_posts.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ def change
       t.string :description
       t.text :content
       t.string :picture
-      t.boolean :publiced
+      t.boolean :publish, default: true
```
Comment (huongnguyenmta) [inline db/migrate/20170627063512_create_posts.rb]: rename `publish` => `is_published`

**Comment by huongnguyenmta** [comment]
commented

**Comment by thanhgiong195** [comment]
ready

**Pull request merged**